cloud-tasks/app/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "postcard-app",
+  "version": "1.0.0",
```

Comment:
@fhinkel has told me to leave off the version property.

Reply:
ack.

Reply:
Note: cannot leave off the version property, latest is to keep it but also `private: true`

Reply:
Why can we not leave off the version property? That should only be relevant if you want to publish something as a library.